### ISSUE TITLE
problem updating to 2.3.2

### ISSUE THREAD

**Issue by sanjeevRJMU1** [bwlewis/irlba#28]
Hi, 

I've been trying to run Monocle2.0 and got stuck with the BLAS/LAPACK error. This issue can be resolved apparently by updating the irlba package to 2.3.2. Unfortunately, I am unable to update to the latest version (attached screenshot). 

Can you help me resolve this error?

Thanks!
Sanjeev

<img width="803" alt="screen shot 2017-11-15 at 2 01 57 pm" src="https://user-images.githubusercontent.com/26235762/32863681-26cdfd30-ca11-11e7-979f-6cba457cdd22.png">

**Comment by sunwu93** [comment]
Hi Sanjeev,

I had the exact same issue as you through monocle. This installation issue is likely with gfortran 6.3.0 installed on your mac. You need to re-install gfortran 6.1.

First you have to delete all gfortran 6.3 files first so you can install from the disk image (disk image wont work otherwise);
sudo -r /usr/local/gfortran
sudo -r /usr/local/bin/gfortran
sudo rm /private/var/db/receipts/com.gnu.gfortran.bom 
/private/var/db/receipts/com.gnu.gfortran.plist

Download here and run the disk image for 6.1
https://gcc.gnu.org/wiki/GFortranBinaries#MacOS

Re-run installation;
install_github("bwlewis/irlba”)

Re-start R 
> library(irlba)

The error from monocle should be resolved

Hope that helps,
Sunny

**Comment by sanjeevRJMU1** [comment]
That resolved the error!!!! Thank you SO much for your help!
Sanjeev

**Comment by bwlewis** [comment]
Thanks so much @sunwu93  for helping out!

**Comment by muktad** [comment]
Is there a tar or zip file for 2.3.2?

**Comment by sejiro** [comment]
Hello, I recently found that I will need to update to 2.3.2 however when I try to install from github I get an error. install.packages() will install irlba but install_github() is having some trouble, see the following:
![screen shot 2018-01-10 at 2 08 20 pm](https://user-images.githubusercontent.com/10656316/34790407-bf39da60-f60f-11e7-8a69-efdcdd26084b.png)

any advice in resolution? Thanks in advance, 

-Sejiro

**Comment by bwlewis** [comment]
Thanks @sejiro  for catching this, a bug!. I'm removing that and re-checking the package now. This was important to catch as I'm trying to get 2.3.2 to cran as soon as possible (just fixed a bunch of issues in prcomp_irlba)...

**Comment by bwlewis** [comment]
Try it now. Please let me know if you find anything else, I hope to send it off perhaps tomorrow to CRAN.

**Comment by sejiro** [comment]
I'll be trying this as soon as I get out of journal club! Thanks!

-Sejiro
________________________________

**Comment by sejiro** [comment]
Still getting the following error, should I go back and update to gfortran 6.3?

![screen shot 2018-01-11 at 10 40 18 am](https://user-images.githubusercontent.com/10656316/34833199-da4d3498-f6bc-11e7-86e7-56615d367a7e.png)

**Comment by bwlewis** [comment]
yes, I think the 6.3 might work now. Again, it's just a guess as I am
unfamiliar with Macintosh...

But good news, the updated package went to cran today and a binary version
will appear for Macs soon (usually within a day or two).



On Jan 11, 2018 10:48, "sejiro" <notifications@github.com> wrote:

Still getting the following error, should I go back and update to gfortran
6.3?

[image: screen shot 2018-01-11 at 10 40 18 am]
<https://user-images.githubusercontent.com/10656316/34833199-da4d3498-f6bc-11e7-86e7-56615d367a7e.png>

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/bwlewis/irlba/issues/28#issuecomment-356968229>, or mute
the thread

**Comment by sejiro** [comment]
With the mac OS X El Capitan binary for 'irlba' going live on CRAN the package once again plays nice with the 'monocle' package I am using. 

That said I am still unsure as to the exact nature of the dyld_stub_binding_helper bug that I am getting trying to install straight from github. If anyone knows a little more about gfortran and mac compilers help with this or a better explanation of the issue would be appreciated.

cheers,
-Sejiro
